app/javascript/react/screens/App/Overview/components/InfrastructureMappingsList/InfrastructureMappingsList.js

```diff
@@ -1,226 +1,551 @@
+                                toggleExpanded={() => {
+                                  this.toggleExpand(mapping, 3);
+                                }}
+                              >
+                                <Icon type="pf" name="catalog" />
+                                <strong>{associatedPlansCount}</strong>&nbsp;
+                                {associatedPlansCount === 1
+                                  ? __('Associated Plan')
+                                  : __('Associated Plans')}
+                              </ListView.Expand>
+                            </ListView.InfoItem>
+                          ) : null
+                        ]}
+                        actions={
+                          inProgressRequestsTransformationMappings.find(
+                            inProgressRequestsTransformationMapping =>
+                              inProgressRequestsTransformationMapping ===
+                              mapping.id
+                          ) ? (
+                            <Icon
+                              type="pf"
+                              className="delete-infra-mapping-icon-disabled"
+                              name="delete"
+                            />
+                          ) : (
+                            <Icon
+                              type="pf"
+                              name="delete"
+                              onClick={e => {
+                                e.stopPropagation();
+                                setMappingToDeleteAction(mapping);
+                                showDeleteConfirmationModalAction();
+                              }}
+                            />
+                          )
+                        }
+                      >
                         <Grid.Row>
-                          <Grid.Col sm={4}>
-                            {clusterName(clusters, item.source_id)}
-                          </Grid.Col>
-                          <Grid.Col sm={4}>
-                            {clusterName(clusters, item.destination_id)}
-                          </Grid.Col>
-                          <Grid.Col sm={4}>
-                            {associatedPlansCount > 0
-                              ? mapping.service_templates.map((plan, id) => (
-                                  <div key={id}>
-                                    <span>{plan.name}</span>&nbsp;&nbsp;
-                                  </div>
-                                ))
-                              : null}
+                          <Grid.Col sm={12}>
+                            {mapping.expandType === 0 && (
+                              <React.Fragment>
+                                <Grid.Row>
+                                  <Grid.Col xs={6}>
+                                    <b>{__('Source Networks')}</b>
+                                  </Grid.Col>
+                                  <Grid.Col xs={6} style={{ paddingLeft: 0 }}>
```

Comment:
[le sigh... one day... ](https://v4-alpha.getbootstrap.com/layout/grid/#no-gutters)

Reply:
those are unique to these only. I guess I can put them in classes if you really want ;)

Reply:
I thought we did this somewhere else... but not finding it.. might not be a bad idea to put em in a class for use elsewhere if/when necessary 🤷‍♂️

Reply:
Sass and friends are happy ;)